examples/plot_run_benchmark.py

```diff
@@ -19,7 +19,8 @@
 
 try:
     save_file = run_benchmark(
-        Benchmark(BENCHMARK_PATH), ['sklearn', 'lightning'],
+        Benchmark(BENCHMARK_PATH), ['sklearn[liblinear]', 'sklearn[saga]',
+                                    'sklearn[newton-cg]', 'lightning'],
```

Comment:
Why is `lightning` not on your figure?

Reply:
It was not installed when I ran `plot_run_benchmark.py` 

I had to because I can't build the doc with `make html`, see upcoming issue